### PR TITLE
Add link to Wiren Board devices firmware change log

### PR DESCRIPTION
**Pull request by KraPete**
Добавил ссылку на страницу с изменениями в прошивке

изображение
изображение

## Summary by CodeRabbit

- **New Features**
	- Enhanced user notifications for firmware and bootloader updates, including links to changelogs for better guidance.
	- Added new labels for software component updates and improved error messages for device configuration issues.
  
- **Bug Fixes**
	- Improved clarity and consistency in error messages related to device updates and configurations.

- **Documentation**
	- Updated changelog to reflect the latest version and provide links to firmware change logs.

### CODE REVIEW

File: app/scripts/i18n/react/locales/en/translations.json [hunk 1]
```diff
@@ -259,9 +259,13 @@
...
-         "new-firmware-bootloader": "New versions are available:<br>     - bootloader {{available_bootloader}}, current {{current_bootloader}};<br>     - firmware {{available_firmware}}, current {{current_firmware}}.",
+         "new-firmware": "There is a new firmware: {{available_firmware}}, current: {{current_firmware}}.",
+         "new-firmware-link": "There is a new firmware: {{available_firmware}}, current: {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Change log</0>",
+         "new-bootloader": "There is a new bootloader: {{available_bootloader}}, current: {{current_bootloader}}.",
+         "new-software-components": "New versions are available:",
+         "new-bootloader-item": "bootloader {{available_bootloader}}, current {{current_bootloader}};",
+         "new-firmware-item": "firmware {{available_firmware}}, current {{current_firmware}}",
+         "new-firmware-item-link": "firmware {{available_firmware}}, current {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Change log</0>.",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/en/translations.json:268]: Fixed

File: app/scripts/i18n/react/locales/ru/translations.json [hunk 2]
```diff
@@ -257,9 +257,13 @@
          "updating-firmware": "Обновление прошивки {{current}} на {{available}}",
          "updating-bootloader": "Обновление загрузчика {{current}} на {{available}}",
          "actual-firmware": "Текущая версия прошивки: {{firmware}} (актуальная)",
-         "new-firmware": "Доступна новая версия прошивки: {{available}}, текущая: {{current}}.",
-         "new-bootloader": "Доступна новая версия загрузчика: {{available}}, текущая: {{current}}.",
-         "new-firmware-bootloader": "Доступны новые версии:<br>     - загрузчик {{available_bootloader}}, текущая {{current_bootloader}};<br>     - прошивка {{available_firmware}}, текущая {{current_firmware}}.",
+         "new-firmware": "Доступна новая версия прошивки: {{available_firmware}}, текущая: {{current_firmware}}.",
+         "new-firmware-link": "Доступна новая версия прошивки: {{available_firmware}}, текущая: {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Список изменений</0>",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/ru/translations.json:261]: Useless

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+wb-mqtt-homeui (2.104.0) stable; urgency=medium
+
+  * Add link to Wiren Board devices firmware change log
```
[the remark this answers is not in the document]
Comment (KraPete) [reply debian/changelog:3]: Useless

File: app/scripts/i18n/react/locales/ru/translations.json [hunk 4]
```diff
@@ -257,9 +257,13 @@
...
+         "new-bootloader": "Доступна новая версия загрузчика: {{available_bootloader}}, текущая: {{current_bootloader}}.",
+         "new-software-components": "Доступны новые версии:",
+         "new-bootloader-item": "загрузчик {{available_bootloader}}, текущая {{current_bootloader}};",
+         "new-firmware-item": "прошивка {{available_firmware}}, текущая {{current_firmware}}",
+         "new-firmware-item-link": "прошивка {{available_firmware}}, текущая {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Список изменений</0>.",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/ru/translations.json]: Useless

File: app/scripts/i18n/react/locales/en/translations.json [hunk 5]
```diff
@@ -259,9 +259,13 @@
          "updating-firmware": "Updating firmware {{current}} to {{available}}",
          "updating-bootloader": "Updating bootloader {{current}} to {{available}}",
          "actual-firmware": "Current firmware version: {{firmware}} (actual)",
-         "new-firmware": "There is a new firmware: {{available}}, current: {{current}}.",
-         "new-bootloader": "There is a new bootloader: {{available}}, current: {{current}}.",
-         "new-firmware-bootloader": "New versions are available:<br>     - bootloader {{available_bootloader}}, current {{current_bootloader}};<br>     - firmware {{available_firmware}}, current {{current_firmware}}.",
+         "new-firmware": "There is a new firmware: {{available_firmware}}, current: {{current_firmware}}.",
+         "new-firmware-link": "There is a new firmware: {{available_firmware}}, current: {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Change log</0>",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/en/translations.json:263]: Useless

File: app/scripts/i18n/react/locales/ru/translations.json [hunk 6]
```diff
@@ -257,9 +257,13 @@
...
-         "new-firmware-bootloader": "Доступны новые версии:<br>     - загрузчик {{available_bootloader}}, текущая {{current_bootloader}};<br>     - прошивка {{available_firmware}}, текущая {{current_firmware}}.",
+         "new-firmware": "Доступна новая версия прошивки: {{available_firmware}}, текущая: {{current_firmware}}.",
+         "new-firmware-link": "Доступна новая версия прошивки: {{available_firmware}}, текущая: {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Список изменений</0>",
+         "new-bootloader": "Доступна новая версия загрузчика: {{available_bootloader}}, текущая: {{current_bootloader}}.",
+         "new-software-components": "Доступны новые версии:",
+         "new-bootloader-item": "загрузчик {{available_bootloader}}, текущая {{current_bootloader}};",
+         "new-firmware-item": "прошивка {{available_firmware}}, текущая {{current_firmware}}.",
+         "new-firmware-item-link": "прошивка {{available_firmware}}, текущая {{current_firmware}}. <0 href=\"https://wirenboard.com/wiki/Changelog?signature={{device_model}}\">Список изменений</0>.",
```
[the remark this answers is not in the document]
Comment (KraPete) [reply app/scripts/i18n/react/locales/ru/translations.json:266]: Useless

**Review by ninedev-i — APPROVED** [review]

**Pull request merged**